File: Python/Human.py
class Human:
    def __init__(self, nik="-", nama="-", jenis_kelamin='-'):
        self.__nik = nik
        self.__nama = nama
        self.__jenis_kelamin = jenis_kelamin
    
    def setNama(self, nama):
        self.__nama = nama

    def getNama(self):
        return self.__nama

    def setNik(self, nik):
        self.__nik = nik

    def getNik(self):
        return self.__nik
	
    def setJenis_kelamin(self, jenis_kelamin):
        self.__jenis_kelamin = jenis_kelamin
	
    def getJenis_kelamin(self):
        return self.__jenis_kelamin
	

File: cpp/Human.cpp
#include <bits/stdc++.h>
using namespace std;
class Human{
	private:
		string nik;
		string nama;
		string jenis_kelamin;
	public:
		Human(){
			this->nik = "-";
			this->nama = "-";
			this->jenis_kelamin = "-";
		}   
		Human(string nik, string nama, string jenis_kelamin){
			this->nik = nik;
			this->nama = nama;
			this->jenis_kelamin = jenis_kelamin;
		}   
		void setNama(string nama){
			this->nama = nama;
		}
		string getNama(){
			return this->nama;
		}
		void setNik(string nik){
			this->nik = nik;
		}
		string getNik(){
			return this->nik;
		}
		void setJenis_kelamin(string jenis_kelamin){
			this->jenis_kelamin = jenis_kelamin;
		}
		string getJenis_kelamin(){
			return this->jenis_kelamin;
		}
};